test/e2e/support/setup-test-framework.js

```diff
@@ -1,23 +1,28 @@
 /**
  * External dependencies
  */
-import puppeteer from 'puppeteer';
+import 'expect-puppeteer';
 
 /**
- * Node dependencies
+ * Internal dependencies
  */
-import { visitAdmin } from './utils';
+import {
+	clearLocalStorage,
+	disablePageDialogAccept,
+	enablePageDialogAccept,
+	visitAdmin,
+} from './utils';
 
-const { PUPPETEER_HEADLESS, PUPPETEER_SLOWMO, PUPPETEER_TIMEOUT } = process.env;
+/**
+ * Environment variables
+ */
+const { PUPPETEER_TIMEOUT } = process.env;
 
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
 
 beforeAll( async () => {
-	global.browser = await puppeteer.launch( {
-		headless: PUPPETEER_HEADLESS !== 'false',
-		slowMo: parseInt( PUPPETEER_SLOWMO, 10 ) || 0,
-	} );
+	enablePageDialogAccept();
```

Comment:
I don't like that we're accepting dialogs globally, I prefer the explicitness we had before, because sometimes we want to test these dialogs

Reply:
We need for the majority of test suites as far as I discovered today. There is `disablePageDialogAccept` method which would be useful in case when you want to handle it yourself. I'm happy to refactor, but then we would have to put it for every test which types something.

Reply:
Ok 👍 It's fine